docs/events.md

```diff
@@ -428,6 +501,54 @@ var GrantParentView = Marionette.View.extend({
 
 [Live example](https://jsfiddle.net/marionettejs/8eq7vca5/)
 
+#### Using `CollectionView`'s `childViewTriggers`
+
+```javascript
+var Mn = require('backbone.marionette');
+
+// The child view fires a custom event, `show:message`
+var ChildView = Mn.View.extend({
+
+  // Events hash defines local event handlers that in turn may call `triggerMethod`.
+  events: {
+    'click .button': 'onClickButton'
+  },
+
+  // Triggers hash converts DOM events directly to view events catchable on the parent.
+  // Note that `triggers` automatically pass the first argument as the child view.
+  triggers: {
+    'submit form': 'submit:form'
+  },
+
+  onClickButton: function () {
+    // Both `trigger` and `triggerMethod` events will be caught by parent.
+    this.trigger('show:message', 'foo');
+    this.triggerMethod('show:message', 'bar');
+  }
+});
+
+// The parent uses childViewEvents to catch the child view's custom event
+var ParentView = Mn.CollectionView.extend({
+
+  childView: ChildView,
+
+  childViewTriggers: {
+    'show:message': 'child:show:message',
+    'submit:form': 'child:submit:form'
```

Comment:
This could be a bit confusing as it's close enough to the default `childview:show:message` events. I find a totally different convention is usually more helpful for new developers to separate the concepts when building applications.